### PR TITLE
bext: add firstSourcegraphUrl to logs

### LINKED ISSUE #32249: Check if it is possible to add "first sourcegraph url" property for browser extension event logs (similar to web event logs)
Browser extension event logs currently don't send "first sourcegraph URL". It is stored in cookies when sent to Sourcegraph GraphQL API. Check if it is possible somehow retrieve it from the background script (see [chrome extension API for reading cookies](https://developer.chrome.com/docs/extensions/reference/cookies/), or also maybe backend implementation to get from cookies directly. If yes, then we need to implement it.

See [slack discussion](https://sourcegraph.slack.com/archives/C01LZKLRF0C/p1646421403801479?thread_ts=1646333370.536989&cid=C01LZKLRF0C)

### PR DESCRIPTION
**Pull request by taras-yemets**
Closes #32249

Adds `firstSourcegraphUrl` to logs when default Sourcegraph URL is used.

## Notes
- tested in Chrome
- telemetry is disabled for Firefox, so logs are not sent at all, see:
  - [client/browser/src/shared/code-hosts/shared/codeHost.tsx](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/browser/src/shared/code-hosts/shared/codeHost.tsx?L1558-1568)
  - [client/browser/src/shared/util/optionFlags.ts](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/client/browser/src/shared/util/optionFlags.ts?L91-123)
 
## Test plan

1. Run  `sg run bext` for _dev_ version or `yarn --cwd client/browser build` for _prod_ one.
2. Set Sourcegrapg URL value to _sourcegraph.com_.
3. Open a file on a code host (for example, [this one](https://github.com/sourcegraph/sourcegraph/blob/main/client/browser/src/browser-extension/options-menu/OptionsPage.tsx)).
4. Open browser extension background page.
5. Perform an action being logged (for example, hover on a token to see a code intel popup).
6. Inspect `logEvent` requests on the network tab of browser extension background page.
7. Ensure that `firstSourceUrl` value **is present** in the request payload.
8. Change Sourcegrapg URL value (for example, to _k8s.sgdev.org_).
9. Repeat steps 5 and 6.
10. Ensure that `firstSourceUrl` value **is not present** in the request payload.

### CODE REVIEW

File: client/browser/src/browser-extension/manifest.spec.json [hunk 1]
```diff
@@ -52,6 +53,13 @@
     "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAvyGmcOkw4cTnhO0bgl3fQLAdv1jZp8T1ZHYI+4d8FgwwVKLYWE+pAuJ/0LrI69ibed4Nnnw5YleB1xCpI+mzB56xfXWboKp6lljevKqWJ5TpJk/Vam3kSSoZwpmJRXnzmcM3qKpL6viUhTfwGmQO6WVTsN4YCx+KWXv97IyF6yDTgd6hwFsvCZY2n1ADgurrQkE6AcJ3kK4xZ14jaHllXEdFcqwh0+Am5qLcIJ1cNo5iFD35exXsjwdQbmpt8sEk5f95pK5FEEbJFmOTguu2fOZycqIoTgoDrbbhT5k9TUogZaN5Lup0Iwh0Cv60i4C1f7IdPrxHuaYmYCfoUezXnQIDAQAB"
   },
   "prod": {
-    "permissions": ["activeTab", "storage", "contextMenus", "https://github.com/*", "https://sourcegraph.com/*"]
+    "permissions": [
+      "cookies",
```
Comment (erzhtor) [inline client/browser/src/browser-extension/manifest.spec.json:57]: @taras-yemets, have you checked how this new permission is requested/shown on the initial install?

Comment (taras-yemets) [reply client/browser/src/browser-extension/manifest.spec.json:57]: I ran browser extension production build locally and wasn't asked about any permissions both in Brave (Chrome) and Firefox.

File: client/browser/src/shared/tracking/eventLogger.tsx [hunk 2]
```diff
@@ -123,11 +124,22 @@ export class EventLogger implements TelemetryService {
         if (userEvent) {
             logUserEvent(userEvent, anonUserId, this.sourcegraphURL, this.requestGraphQL)
         }
+
+        const firstSourceURL = isDefaultSourcegraphUrl(this.sourcegraphURL)
+            ? (
+                  await (isBackground ? browser.cookies.get : background.getCookie)({
```
Comment (erzhtor) [inline client/browser/src/shared/tracking/eventLogger.tsx:130]: 👍

**Review by erzhtor — APPROVED** [review]
Neat, LGTM! 👍

**Pull request merged**